### PR TITLE
Added backup and restore methods to Drill storage plugins

### PR DESCRIPTION
**Pull request by denysord88**

### CODE REVIEW

**Comment by denysord88** [comment]
@agozhiy @Atri0 please review the PR.

**Comment by Atri0** [comment]
LGTM

File: src/test/java/org/apache/drill_web_test_framework/rest_api/data/RestBaseSteps.java [hunk 1]
```diff
@@ -23,14 +23,37 @@
...
+import static io.restassured.RestAssured.given;
 
+public final class RestBaseSteps {
   private static boolean storagePluginsSetupComplete = false;
+  private static HashMap<String, String> storagePluginsOriginal = new HashMap<>();
 
   private RestBaseSteps() {
   }
 
+  public static void backupPlugin(String pluginName) {
```
Comment (agozhiy) [inline src/test/java/org/apache/drill_web_test_framework/rest_api/data/RestBaseSteps.java]: Should be moved to RestStorageSteps.

File: src/test/java/org/apache/drill_web_test_framework/rest_api/data/RestBaseSteps.java [hunk 2]
```diff
@@ -23,14 +23,37 @@
...
+    storagePluginsOriginal.putIfAbsent(pluginName, getStoragePlugin(pluginName));
+  }
+
+  public static void restorePlugins() {
```
Comment (agozhiy) [inline src/test/java/org/apache/drill_web_test_framework/rest_api/data/RestBaseSteps.java]: Should be moved to RestStorageSteps.

File: src/test/java/org/apache/drill_web_test_framework/rest_api/data/RestBaseSteps.java [hunk 3]
```diff
@@ -23,14 +23,37 @@
...
+    storagePluginsOriginal.forEach((k, v) ->
+        RestStorageSteps.updateStoragePlugin(v, RestSecuritySteps.getDefaultSession())
+    );
+  }
+
+  private static String getStoragePlugin(String pluginName) {
```
Comment (agozhiy) [inline src/test/java/org/apache/drill_web_test_framework/rest_api/data/RestBaseSteps.java]: Should be moved to RestStorageSteps.

**Review by agozhiy — CHANGES_REQUESTED** [review]

File: src/test/java/org/apache/drill_web_test_framework/rest_api/data/RestBaseSteps.java (hunk 1 above)
Comment (denysord88) [reply src/test/java/org/apache/drill_web_test_framework/rest_api/data/RestBaseSteps.java]: Done

File: src/test/java/org/apache/drill_web_test_framework/rest_api/data/RestBaseSteps.java (hunk 2 above)
Comment (denysord88) [reply src/test/java/org/apache/drill_web_test_framework/rest_api/data/RestBaseSteps.java]: Done

File: src/test/java/org/apache/drill_web_test_framework/rest_api/data/RestBaseSteps.java (hunk 3 above)
Comment (denysord88) [reply src/test/java/org/apache/drill_web_test_framework/rest_api/data/RestBaseSteps.java]: Done

**Comment by denysord88** [comment]
@agozhiy please review changes.

**Comment by agozhiy** [comment]
+1.

**Review by agozhiy — APPROVED** [review]

**Pull request merged**